reggie_config/load/init.yaml

```diff
@@ -1,13 +1,6 @@
-__: merge-last
+__: merge-first
```

Comment:
I updated this file to match what's in the staging environment, just on general principle.  The files in the "load" environment are over a year old, and it seemed better to have files which match our other stacks.  I checked and didn't find anything that looked like it needed to stay a particular way.